symbolic-debuginfo/src/dwarf.rs

```diff
@@ -825,7 +825,11 @@ impl<'d, 'a> DwarfUnit<'d, 'a> {
             let function = Function {
                 address: function_address,
                 size: function_size,
-                name: Name::with_language(name.unwrap_or_default(), self.language),
+                name: Name::new(
+                    name.unwrap_or_default(),
+                    NameMangling::Unknown,
```

Comment:
This we can infer by knowing where the name comes from. This will require a slightly larger refactor:

 - If the name comes from the symbol table, then it is `Mangled`
 - If the name comes from `DW_AT_linkage_name` or `DW_AT_MIPS_linkage_name`, it is `Mangled`
 - If the name comes from `DW_AT_name`, then it is `Unmangled`, to my recollection. This we have to verify again, though.